types/attribute.go

```diff
@@ -40,6 +53,20 @@ func (m Attribute) SubsetOf(rhs Attribute) bool {
 	return false
 }
 
+func (attr Attributes) Validate() error {
```

Comment:
Is it possible to have an empty name here?

I think we should also restrict the names to be `[a-zA-Z][a-zA-Z_-]{0,32}` or something, too.

Reply:
attributes in sdl is a map so it will fail if key is empty, but checking name is a good point

Reply:
`scheiße` 32 is bit too short, `testutil` generates long keys 🤣